app/serializers/billing-detail.js

```diff
@@ -0,0 +1,8 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  extract: function(store, type, payload, id, requestType) {
+    this.__requestType = requestType; // used by `extractMeta`
```

Comment:
Can this comment be elaborated upon? Used by `extractMeta` for what? It seems unfortunate to set a `__doubleUnderscorePrivate` value.